app/lib/frontend/email_sender.dart

```diff
@@ -141,11 +143,14 @@ class _GmailSmtpRelay implements EmailSender {
     _logger.info('Sending email: $debugHeader...');
     try {
       await retry(
-        () async => send(
-          _toMessage(message),
-          await _getSmtpServer(),
-          timeout: Duration(seconds: 15),
-        ),
+        () async {
+          final c = await _getConnection();
+          try {
+            await c.connection.send(_toMessage(message));
```

Comment:
We might not want to reuse the connection if we get a `TimeoutException` or `IOException` or `SmtpClientAuthenticationException` or `MailerException` (not sure about the last one)

Reply:
Updated with a `_forceReconnect` flag to always stop the current connection and create a new one when there was any issue with it. It may be that we could do better and sometimes continue using it, but I think it is better to keep it simple.